utils/utilityApi.js

```diff
@@ -7,15 +7,13 @@ A set of low-level functions that gives us the details we need from the Utility
 import fetch from 'node-fetch';
 import moment from 'moment';
 
-const API_KEY = process.env.UTILITY_API_KEY;
-const EBCERebateString = 'Credited to (Debited from) NEM Balance';
-
-// Base URL for Utility API BILLS
+// Constants
 const billsBaseUrl = 'https://utilityapi.com/api/v2/bills';
+const EBCERebateString = 'Credited to (Debited from) NEM Balance';
```

Comment:
is there a reason you defined these here instead of putting them in the Constants.js file?

Reply:
I think that our constants file would be most productive being limited to things that could be considered "Parameters" or "Settings" for the app, in the sense that they are things we might want to tweak quickly without revisiting the code. 

These two variables are constants, but they are very tied to how the calls to utility API are queried. I'm assuming that if one of these changed, we'd have to rewrite the code in here too, so I grouped them together.

honestly..just kinda personal preference. we can move them if we want